.github/scripts/prepare_JREs.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+URL_JDK8="https://api.adoptium.net/v3/binary/latest/8/ga/linux/x64/jre/hotspot/normal/eclipse"
+URL_JDK17="https://api.adoptium.net/v3/binary/latest/17/ga/linux/x64/jre/hotspot/normal/eclipse"
+
+mkdir -p JREs
+pushd JREs
+
+wget --content-disposition "$URL_JDK8"
+wget --content-disposition "$URL_JDK17"
+
+for file in *;
+do
+    mkdir temp
+
+    # Handle OpenJDK17 archive
```

Comment:
This whole thing could probably be condensed into a for loop (`for n in 8 17`). The two lines that are different can be put in an if statement

Reply:
No, it needs to read the file names to handle regex for the version numbers. We're pulling latest builds, not a pinned version. I de-duped the code, regardless.